sdk/src/main/scala/com/horizen/account/state/ForgerStakeMsgProcessor.scala

```diff
@@ -306,7 +306,15 @@ case class ForgerStakeMsgProcessor(params: NetworkParams) extends AbstractFakeSm
 
     if (isGenesisScCreation) {
       // no gas paid here
+
+      // increase the balance of the "forger stake smart contract” account
+      view.addBalance(fakeSmartContractAddress.address(), stakedAmount).get
+
+      // TODO
+      //view.addLog(new EvmLog concrete instance) // EvmLog will be used internally
```

Comment:
no logs needed here. since the balance was updated by McBlockRefData not but Tx in the block